src/types/markers.js

```diff
@@ -26,22 +26,20 @@ export type CauseBacktrace = {
 };
 
 /**
- * These markers have a start and end time.
+ * These markers don't have a start and end time. They work in pairs, one
+ * specifying the start, the other specifying the end of a specific tracing
+ * marker.
  */
 export type PaintProfilerMarkerTracing_Gecko = {
   type: 'tracing',
   category: 'Paint',
-  startTime: Milliseconds,
```

Comment:
Do you happen to know the history of why these are wrong?

Reply:
I believe this is a mistake, introduced in https://github.com/devtools-html/perf.html/pull/434, by mimicry with other payloads.